etl/stock_hard_delete_pattern.py

```diff
@@ -0,0 +1,64 @@
+from sqlalchemy import create_engine, text
+from datetime import datetime
+from io import StringIO
+import os
+import boto3
+import pandas as pd
+
+
+# Connect MySQL db
+db_user = "admin"
+db_password = "password"
+db_host = "my-db-instance.cxjkxlbvg4uh.ap-northeast-1.rds.amazonaws.com"
+db_name = "stock_db"
+
+db_url = os.environ['DB_URL']
+
+engine = create_engine(db_url)
+
+conn = engine.connect()
+
+# Detect updated records and upload into s3
+etl = '''
+            SELECT 
+                new.*
+            FROM stock AS new
+            LEFT ANTI JOIN stock_staging AS old
+                ON new.stock_id = old.stock_id AND new.company = old.company AND new.category = old.category AND new.price = old.price
+      '''
+updated_records = conn.execute(text(etl)).fetchall()
```

Comment:
Please get old data at day (n-2) from s3, then join with (n-1) (get from RDBMS) to find updated records